### PR TITLE
Chart: Support multiple service account issuers, fix test values filenames, allow `global.connectivity.network.pods.nodeCidrMaskSize` in schema

### PR DESCRIPTION
**Pull request by AndiDog**
### What this PR does / why we need it

Port of https://github.com/giantswarm/cluster-aws/pull/814 to `main` branch.

Let's see if CI succeeds like this. The CI files weren't used before by mistaken naming.

Depends on https://github.com/giantswarm/cluster/pull/341

### Checklist

- [x] Updated CHANGELOG.md.

### Trigger E2E tests

not needed

### CODE REVIEW

File: helm/cluster-aws/templates/_control_plane.tpl [hunk 1]
```diff
@@ -50,14 +50,17 @@ subnet:
     {{- end }}
 {{- end }}
 
+{{- define "service-account-issuers-comma-separated" }}
+{{- range $serviceAccountIssuerIndex, $serviceAccountIssuer := .Values.cluster.providerIntegration.controlPlane.kubeadmConfig.clusterConfiguration.apiServer.serviceAccountIssuers }}
+{{- if gt $serviceAccountIssuerIndex 0 }},{{- end -}}
+{{ regexReplaceAll "^(http://|https://)" (include "cluster.internal.controlPlane.kubeadm.clusterConfiguration.apiServer.serviceAccountIssuer" (dict "Values" $.Values "Release" $.Release "serviceAccountIssuer" $serviceAccountIssuer)) "" }}
```
Comment (fiunchinho) [inline helm/cluster-aws/templates/_control_plane.tpl:56]: Why do we have to pass the `Values` and the `Release`?

Comment (AndiDog) [reply helm/cluster-aws/templates/_control_plane.tpl:56]: IIRC it uses those to form some `.metadata.name = {{ ... }}` by `include`ing other template functions.

**Review by fiunchinho — APPROVED** [review]

**Comment by AndiDog** [comment]
/run cluster-test-suites TARGET_SUITES=./providers/capa/standard

**Comment by AndiDog** [comment]
Force-merging because the expected build version `<tag>-<commit>` differs between `abs`/`cluster-test-suites` and I don't have the brains to work around that now. Manually creating a WC worked.

**Pull request merged**